lib/Conversion/CalyxToFSM/CalyxToFSM.cpp

```diff
@@ -268,6 +268,69 @@ LogicalResult CompileFSMVisitor::visit(StateOp currentState, EnableOp enableOp,
   return success();
 }
 
+// CompileInvoke is used to convert invoke operations to group operations and
+// enable operations.
+class CompileInvoke {
+public:
+  CompileInvoke(ComponentOp component, OpBuilder builder)
+      : component(component), builder(builder) {}
+  void compile();
+
+private:
+  void lowerInvokeOp(InvokeOp invokeOp);
+  ComponentOp component;
+  OpBuilder builder;
+  size_t groupNameTail = 0;
```

Comment:
Missing comment. Also, reading the code, it doesn't seem like this needs to be a member variable.

Reply:
` it doesn't seem like this needs to be a member variable`, I don't really understand what you're saying here.

Reply:
I was suggesting to make this a member variable since it is used to generate unique group names. This way, the unique counter is reused across multiple calls to lowerInvokeOp, so the loop that's checking for name uniqueness usually finds a unique name on the first try.

Reply:
I am waiting for the result of your discussion, so it is a little late to make the changes.

Reply:
@linuxlonelyeagle I think this discussion is resolved, and the current implementation is fine. @mortbopet was asking why this `groupNameTail` is defined here as a member variable of the `CompileInvoke` class, and I explained why I had suggested to do it like this.